src/main/java/com/greenfoxacademy/opal/kalendaryo/kalendaryo/model/entity/SharingOptions.java

```diff
@@ -0,0 +1,58 @@
+package com.greenfoxacademy.opal.kalendaryo.kalendaryo.model.entity;
+
+public class SharingOptions {
+
+    Boolean title;
+    Boolean organizer;
+    Boolean location;
+    Boolean attendants;
+    Boolean description;
```

Comment:
@matheb you can use `boolean`-s instead of `Boolean`-s, they have `false` default value, so no need for constructor, if they are all public, no need for getters and setters